images/runtime/node/Dockerfile.template

```diff
@@ -22,6 +22,13 @@ ENV ORYX_AI_INSTRUMENTATION_KEY=${AI_KEY}
 RUN mkdir -p /node_modules \
  && chmod 777 /node_modules
 
+RUN set -e \
+ && npm_ver=$(npm --version) \
+ && if [ ! "$npm_ver" = "${npm_ver#6.}" ]; then \
```

Comment:
what is the purpose of this if?

Reply:
couldn't you use `[ ${npm_ver:0:2} = "6." ]` ?

Reply:
Since this change is in the docker template file, it is being used in some Node versions which do not use version `6` of npm, so had to filter

Reply:
> couldn't you use [ ${npm_ver:0:2} = "6." ] ?

Sure, I will give it a try

Reply:
Yeah I gave it a try but it does not work and seems like that is because by default `RUN` instruction in dockerfile runs using `/bin/sh` and not bash (when i change the shell to bash though it works, but there are 2 approaches here: one inline 'bash' and other using `SHELL`..both are not great IMO), so I will leave it as it is now.